benchmarks/defaults.hpp

```diff
@@ -32,7 +32,10 @@ auto constexpr MULTIPLICITY  = 8;
 auto constexpr MATCHING_RATE = 0.5;
 auto constexpr MAX_NOISE     = 3;
 auto constexpr SKEW          = 0.5;
+auto constexpr BATCH_SIZE    = 1'000'000;
```

Comment:
Could we use a larger batch size to reduce the benchmark time a bit?

Reply:
I don't know what reasonable values we should set. Let's check with @Nicolas-Iskos so we can address this in a follow-up PR.